spec/activerecord-bitemporal/bitemporal_spec.rb

```diff
@@ -1052,6 +1072,8 @@ class EmployeeWithUniquness < Employee
     it { expect { subject }.to change { Employee.ignore_valid_datetime.within_deleted.count }.by(1) }
     it { expect { subject }.to change(employee, :swapped_id).from(@swapped_id_before_destroy).to(kind_of(Integer)) }
     it { expect { subject }.to change(employee, :swapped_id_previously_was).from(kind_of(Integer)).to(@swapped_id_before_destroy) }
+    xit { expect { subject }.to change(employee, :changes).to(be_empty) }
```

Comment:
Perhaps the expected is `be_empty`, but unfortunately that is not the case at this time. This will be fixed in another PR.